pkg/radarr/radarr.go

```diff
@@ -72,6 +72,7 @@ type Release struct {
 	PublishDate      string `json:"publishDate"`
 	DownloadClientId int    `json:"downloadClientId,omitempty"`
 	DownloadClient   string `json:"downloadClient,omitempty"`
+	Freeleech        bool   `json:"freeleech,omitempty"`
```

Comment:
Are you sure this is supported? Can't find anything about it in the [Radarr API spec](https://radarr.video/docs/api/#/ReleasePush/post_api_v3_release_push).

Reply:
Talked to bogdan in Servarr and this field is not supported, nor is `indexerFlags` at the moment. They are working on refactoring changes and it will likely be in radarr v6.

Reply:
Supported can mean different things here. It is supported in the sense that *arr accept the respective request, but it's not supported as the Freeleech is ignored for now. Would propose to remove it and work on a bigger webhooks overhaul instead